spec/classes/jira_facts_spec.rb

```diff
@@ -10,8 +10,7 @@
 
         regexp_pe = %r{^#\!/opt/puppet/bin/ruby$}
         regexp_oss = %r{^#\!/opt/puppetlabs/puppet/bin/ruby$}
-        regexp_url = %r{http://127.0.0.1\:8080/rest/api/2/serverInfo}
-        pe_external_fact_file = '/etc/puppetlabs/facter/facts.d/jira_facts.rb'
+        regexp_url = Regexp.new(Regexp.escape('http://localhost:8080/rest/api/2/serverInfo'))
```

Comment:
Just a thought: should it be
```suggestion
        url = 'http://localhost:8080/rest/api/2/serverInfo'
```

And then later `with_content(Regexp.new(Regexp.escape(url)))`